MParT/MonotoneComponent.h

```diff
@@ -1023,4 +1073,26 @@ class MonotoneComponent : public ConditionalMapBase<MemorySpace>
 };
 
 } // namespace mpart
+
+#if defined(MPART_HAS_CEREAL)
+CEREAL_FORCE_DYNAMIC_INIT(mpart1)
```

Comment:
would I be a villain if I suggested we name these like `mpartInitMapFactory1`?

Reply:
That's fine.  We can actually put any name we want here as long as it matches what's in the header file.  It also does nothing but produce some code that causes the file to be tracked by Cereal.